protocol/0006-POSI-positions_core.md

```diff
@@ -37,6 +37,7 @@
 
 - [ ] Maintains separate position data for each market a trader is active in (<a name="0006-POSI-026" href="#0006-POSI-026">0006-POSI-026</a>)
 - [ ] If there is either one or more of the position record's fields is non zero (i.e. open position size, active buy order size, active sell order size), the position record exists. (<a name="0006-POSI-027" href="#0006-POSI-027">0006-POSI-027</a>)
+- [ ] Open positions are closed for parties who are closed out during trading.(<a name="0006-POSI-028" href="#0006-POSI-028">0006-POSI-028</a>)
```

Comment:
This sounds a bit like a tautology. 

What you want to be checking is that a distressed trader gets closed out, assuming there is volume on the book. That's [0012-POSR-002](https://github.com/vegaprotocol/specs/blob/master/protocol/0012-POSR-position_resolution.md#0012-POSR-002)).